File: anti-thief-new/app/src/main/java/com/emopass/antitheftalarm/utils/NetworkUtils.java
package com.emopass.antitheftalarm.utils;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;
import com.emopass.antitheftalarm.App;

public class NetworkUtils {
    private static Boolean isOnline(Context context) {
        ConnectivityManager connectivityManager = (ConnectivityManager) (context.getSystemService(Context.CONNECTIVITY_SERVICE));
        NetworkInfo info = null;
        if(connectivityManager != null) {
            info = connectivityManager.getActiveNetworkInfo();
        }
        return info != null && info.isConnected();
    }

    public static Boolean isInternetAvailable() {
        return isOnline(App.getInstace().getApplicationContext());
    }
}

File: findmyphone/app/src/main/java/com/example/claptofindphone/utils/NetworkUtils.kt
package com.example.claptofindphone.utils

import android.content.Context
import android.net.ConnectivityManager
import com.example.claptofindphone.presenter.ClapToFindApplication


object NetworkUtils {

    fun isOnline(context: Context): Boolean {
        val connectivityManagerInstance =
            context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        val activeNetworkInfoValue = if (connectivityManagerInstance != null) {
            connectivityManagerInstance.activeNetworkInfo
        } else null
        return activeNetworkInfoValue != null && activeNetworkInfoValue.isConnected
    }

    fun isInternetAvailable(): Boolean {
        return isOnline(ClapToFindApplication.getContext())

    }
}